### PR TITLE
upgrade gke initialClusterVersion to 1.17 as 1.16 is no more available

### PR DESCRIPTION
**Pull request by echarles**
Fixes https://github.com/upbound/platform-ref-gcp/issues/5

This PR upgrade the GKE `initialClusterVersion` to 1.17 as 1.16 is no more available.

### CODE REVIEW

**Comment by stevendborrelli** [comment]
Are you ok with going to 1.18 instead? I believe we have to fix some auth issues before we go to 1.19.

**Comment by echarles** [comment]
I have only tested the configuration with  `initialClusterVersion: 1.17`. Will try out 1.18 and bump if it works find.

Yes, 1.19 needs some fixes for auth.

**Review by jbw976 — APPROVED** [review]
Thank you for finding this issue and providing a tested fix for it @echarles! great stuff! :)

I think we should take this fix in now (since it was tested/working) and then we'll continue to invest towards 1.18 along with disabling private nodes with #8, which needs a bit more debugging/testing before it's ready.

Thanks again @echarles, really appreciate your contributions recently! 💪

**Comment by jbw976** [comment]
oh and @echarles could you please add a DCO sign-off to your commit? just realized this branch didn't have that auto check on it, but we should make sure that's done before merging.

From the general crossplane contributing docs: https://github.com/crossplane/crossplane/blob/d04a059fe341a941760a3a94ef07085ae7158365/CONTRIBUTING.md#contributing-code

> Ensure each of your commits is signed-off in compliance with the Developer Certificate of Origin by using git commit -s

**Comment by echarles** [comment]
Sure. let me re-setup my gpg key on my env. Will do that my tomorrow.

**Comment by jbw976** [comment]
thanks @echarles! to be clear, this isn't cryptographic signing that needs a key, this is just the git sign-off you get with a `git commit -s`.

since you've already made the commit, i think this is what you need:
```
git commit -s --amend
```

More details: https://stackoverflow.com/a/13457395

**Comment by echarles** [comment]
Thx for the help @jbw976. I have signed https://github.com/upbound/platform-ref-gcp/pull/6/commits/e484e23e8a7146df525bcdd528fbc71d073b6f00

Checks are now green.

**Comment by echarles** [comment]
Probably superseded by https://github.com/upbound/platform-ref-gcp/pull/8 which bumps to 1.18

**Comment by jbw976** [comment]
perfect - this looks good @echarles! really appreciate your efforts on this contribution! 💪

**Review by jbw976 — APPROVED** [review]

**Pull request merged**
